xml/systemd.xml

```diff
@@ -1475,7 +1473,72 @@ UMask=007</screen>
     For further information, refer to the man pages that can be
     evoked with the <command>man 1 systemctl</command> command.
    </para>
-   <sect3 xml:id="systemd-xinetd-conversion">
+  </sect2>
+  
+  <sect2 xml:id="sec-boot-systemd-custom-drop-in">
+   <title>Creating drop-in files</title>
+   <para>
+    If you only want to add a few lines to a configuration file or modify a
+    small part of it, you can use so-called <quote>drop-in</quote> files.
+    Drop-in files let you extend the configuration of unit files without having
+    to edit or override the unit files themselves.
+   </para>
+   <para>
+    For example, to change one value for the <replaceable>FOOBAR</replaceable>
+    service located in
+    <filename>/usr/lib/systemd/system/<replaceable>FOOBAR.SERVICE</replaceable></filename>,
+    proceed as follows:
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Create a directory called
+      <filename>/etc/systemd/system/<replaceable>FOOBAR</replaceable>.service.d/</filename>.
+     </para>
+     <para>
+      Note the <literal>.d</literal> suffix. The directory must otherwise be
+      named like the service that you want to patch with the drop-in file.
+     </para>
+    </step>
+    <step>
+     <para>
+      In that directory, create a file
+      <filename><replaceable>your_modification</replaceable>.conf</filename>.
+     </para>
+     <para>
+      Make sure it only contains the line with the value that you want to
+      modify.
+     </para>
+    </step>
+    <step>
+     <para>
+      Save your changes to the file. It will be used as an extension of the
+      original file.
```

Comment:
```
      It will be used as an extension of the
      original file.
```

I think this sentence is redundant. By now, it's clear what drop-in files are for.

Reply:
sentence removed